_members/2020/spring.md

```diff
@@ -31,18 +31,18 @@ members:
 - Sean Zhang
 - Teddy Nguyen
 - Tim Bostanche
-- Safwat Rahman
-- Max Albers
-- Franklin Wang
-- Parker Jorenby
-- Kashin Shah
-- Connor Dyjach
-- Abhishek Mhatre
-- Girik Soni
-- George Kimbrough
-- Robert Miller
-- Bhavt Sai Sunkara
-- Nagon Taronvanchni
-- Yun Su Um (Cody)
+#- Safwat Rahman
+#- Max Albers
+#- Franklin Wang
+#- Parker Jorenby
+#- Kashin Shah
+#- Connor Dyjach
+#- Abhishek Mhatre
+#- Girik Soni
+#- George Kimbrough
+#- Robert Miller
+#- Bhavt Sai Sunkara
+#- Nagon Taronvanchni
+#- Yun Su Um (Cody)
```

Comment:
Should these names be present or not?

If not, can you delete these lines? Commented out code is confusing for other viewers.